translations/translations.json

```diff
@@ -301,12 +302,14 @@
     "en": {
       "languageNotAvailable": "Sorry, but your language is not available here.",
       "chooseALanguage": "Choose a language.",
-      "lastUpdate": "Last update:"
+      "lastUpdate": "Last update:",
+      "loading": "Loading ..."
     },
     "es": {
       "languageNotAvailable": "Tu idioma no está disponible aquí.",
       "chooseALanguage": "Selecciona un idioma.",
-      "lastUpdate": "Última modificación:"
+      "lastUpdate": "Última modificación:",
+      "loading": "Cargando ..."
```

Comment:
Wo kommen die Übersetzungen her? Ich habe jetzt "Loading" in englisch, deutsch und spanisch drin.

Reply:
We give this file (or a generated .csv) more or less regularly to a translation agency which fills any missing items. So you only have to include translations you are sure about.